apps/include/pcl/apps/manual_registration.h

```diff
@@ -105,7 +107,14 @@ class ManualRegistration : public QMainWindow {
   void
   DstPointPickCallback(const pcl::visualization::PointPickingEvent& event, void*);
 
+  void
+  refreshView();
+
 protected:
+
+  PCLQVTKWidget* qvtk_widget_src_;
```

Comment:
Please add a comment if they are managed by someone else, else use a unique_ptr

Reply:
I changed so we can use the .ui files anyways. So Qt hierachi will be responsible.